### DISCUSSION TITLE
Pin Multiplexing with Seeed Studio XIAO ESP32S3 (Sense) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#142] [Q&A]
# Pin Multiplexing with Seeed Studio XIAO ESP32S3 (Sense) | Seeed Studio Wiki

Pin multiplexing with Seeed Studio XIAO ESP32S3.

https://wiki.seeedstudio.com/xiao_esp32s3_pin_multiplexing/

**Comment by VR-Architect** [comment]
Please add example for using the Sense camera. And using the Sense camera with 3 PWM servos. Thx

**Reply by MatthewJeffson** [reply]
Hi, we will publish the example with onboard components as scheduled. <br/> Meanwhile, we are also encouraging users publish their own. The detail about it still will be published soon :) Thank you for your interests in our product!

**Comment by mgsb** [comment]
Is there a schematic available? Or a pinout for all board peripherals? Which pins is the SD Card connector using, for example? Thanks, Mark

**Comment by mgsb** [comment]
I did get SD card working with:
* MOSI: 9
* MISO: 8
* SCK: 7
* CS: 21
Note that 21 is also the LED, so it flickers while accessing the SD card. See the SD card File System example: https://wiki.seeedstudio.com/xiao_esp32s3_sense_filesystem/

**Reply by MatthewJeffson** [reply]
Nice!

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by lagunacomputer** [comment]
Hello, what are GPIO#'s or D#'s for the two small LED lights (below the tiny buttons) on the main esp32s3 xiao ?

**Reply by easyway0301** [reply]
UserLed = Pin(21, Pin.OUT)
